README.md

```diff
@@ -62,6 +62,7 @@ until the SDK is marked stable.
     - [Activity Worker Shutdown](#activity-worker-shutdown)
     - [Activity Concurrency and Executors](#activity-concurrency-and-executors)
     - [Activity Testing](#activity-testing)
+  - [Ractors](#ractors)
```

Comment:
??

Reply:
I forgot to auto-generate this TOC entry when I made this section last